developer-docs-site/docs/community/contributions/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Community Highlights"
+slug: "community-highlights"
+---
+
+# Aptos Community Highlights
+
+This page shines a light on exemplary contributions to Aptos from our community members. These enhancements can be documentation, features, or both. These highlights complement both the standalone [Ecosystem Projects](https://github.com/aptos-foundation/ecosystem-projects) and [External Resources](../external-resources.md) from the community.
+
+To add your own contribution here, first submit a new page to Aptos.dev or a product feature in the [`aptos-core`](https://github.com/aptos-labs/aptos-core) repository. Then add a link to the new documentation or code from the table below.
+
+The source materials for both code and docs live in GitHub. See [Contribute](../index.md) for ways to make updates.
+
+| Contribution | Description | Author | Date added/updated |
+| --- | --- | --- | --- |
+| [Multisig Python SDK tooling and tutorial](https://github.com/aptos-labs/aptos-core/pull/6788) | This tutorial introduces assorted [K-of-N multi-signer authentication](../../concepts/accounts.md#multi-signer-authentication) operations | [alnoki](https://github.com/alnoki) | 2023-03-06 |
+| [Remix IDE plugin for Move](https://github.com/aptos-labs/aptos-core/pull/6458) | This tutorial describes a plugin that can develop you develop Aptos Move modules with the Remix IDE. | [0xhsy](https://github.com/0xhsy) | 2023-03-02 |
```

Comment:
do we want to actively link to PRs?

Reply:
No, I was hoping at least one would land by now. :-)

I have moved this and the TODO that follows to the issue:
https://github.com/aptos-labs/aptos-core/issues/6311